### ISSUE TITLE
`Page` `SecondaryAction` + `Pagination` color updates

### ISSUE THREAD

**Issue by sarahill** [Shopify/polaris-react-archive#10354]
Update `disabled` state background color to use `fill` color tokens:

**Disabled state:**

`bg-fill-disabled`

![Image](https://github.com/Shopify/polaris/assets/8629173/0d2ef8c1-c2f2-4e53-8863-0147b690c7da)

**Comment by sophschneider** [comment]
I think this will be covered by the automatic migration `--p-color-bg-transparent-disabled-experimental` -> `--p-color-bg-fill-disabled`. `SecondaryActions` uses `--p-color-bg-transparent-disabled-experimental`  and `Pagination` uses a disabled button which uses `--p-color-bg-transparent-disabled-experimental`. @sarahill lmk if theres any other considerations for this issue!

https://github.com/Shopify/polaris/pull/10579/files#diff-66225532fab9ef9c681f1969b6bfab9480ea71cb751c93c979f11e0749f4ace3R82-R83

**Comment by sarahill** [comment]
@sophschneider I was thinking the mapping to transparent disabled was to `bg-surface-disabled` so that's why I called out this one since it's `bg-fill-disabled` instead... ~but maybe it catches both? I saw both in the map but I assumed it would just connect to one or the other.~


Oh I see what happened:
`bg-transparent-disabled` -> `bg-fill-disabled`
`bg-disalbed` -> `bg-surface-disabled`

Ok yea I think this makes sense. It won't be perfect but neither mapping would be and visually they'll be the same at least. Thanks for clarifying that! 

Based on that I think this issue can be closed then.
